<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Store_Edit_Two</class>
 <widget class="QWidget" name="Store_Edit_Two">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>365</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>思源黑体 CN Medium</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>思源黑体 CN Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>支持最高五种交换物，如果所需交换物小于五种请把后面的留空。</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>#</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>交换物</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>数量</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="exchange_edit_1"/>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="price_edit_1"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="exchange_edit_2"/>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="price_edit_2"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="exchange_edit_3"/>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="price_edit_3"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="exchange_edit_4"/>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="price_edit_4"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="exchange_edit_5"/>
     </item>
     <item row="6" column="2">
      <widget class="QLineEdit" name="price_edit_5"/>
     </item>
     <item row="7" column="0" colspan="3">
      <widget class="QPushButton" name="write_multi_price">
       <property name="text">
        <string>写入</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
